docs/src/account/address.md

```diff
@@ -0,0 +1,63 @@
+# Address
+
+> [!Note]
+> A human-readable identifier for `Account`s or public keys.
+
+
+## Purpose
+
+An address is a unique identifier that facilitates sending and receiving of [notes](../note.md). It has essentially two purposes that are explained in this section.
+
+### Access control for notes
+
+An address determines who is allowed to consume a note. In this sense, it acts as an access-control mechanism.
```

Comment:
Is it more accurate to say that an address **expresses** who is allowed to consume the note? The fact that determines it would be the interface or `PublicKey` or am I mistaken?

Reply:
Yes, great feedback. I wasn't too happy with "determines", and "expresses" is a better wording - thanks!